elephant/test/test_phase_analysis.py

```diff
@@ -370,5 +373,266 @@ def testPhaseLockingValue_raise_Error_if_trial_lengths_are_different(self):
+                  np.cos(2 * 36 * np.pi * times + np.pi / 2) +
+                  np.cos(2 * 52 * np.pi * times) +
+                  np.cos(2 * 100 * np.pi * times) +
+                  np.cos(2 * 70 * np.pi * times + np.pi/2 + np.pi*(i % 2)) +
+                  np.random.normal(loc=0.0, scale=1, size=len(times))
+                  for i in range(ntrial)]
+        lfps_1 = np.stack(lfps_1, axis=0)
+
+        lfps_2 = [np.cos(2 * 16 * np.pi * times) +
+                  np.cos(2 * 36 * np.pi * times) +
+                  np.cos(2 * 52 * np.pi * times + np.pi/2) +
+                  np.cos(2 * 100 * np.pi * times + np.pi/2) +
+                  np.cos(2 * 70 * np.pi * times) +
+                  np.random.normal(loc=0.0, scale=1, size=len(times))
+                  for _ in range(ntrial)]
+        lfps_2 = np.stack(lfps_2, axis=0)
+
+        # save artificial LFP-dataset to .mat files
+        mdic_1 = {"lfp_matrix": lfps_1, "time": times, "sf": srate}
+        mdic_2 = {"lfp_matrix": lfps_2, "time": times, "sf": srate}
+        filename1_a = os.path.sep.join(['cross_testing_scripts',
+                                        'artificial_LFPs_1.mat'])
+        filename2_a = os.path.sep.join(['cross_testing_scripts',
+                                        'artificial_LFPs_2.mat'])
+
+        savemat(filename1_a, mdic_1)
+        savemat(filename2_a, mdic_2)
+
+    def setUp(self):
+        self.tolerance = 1e-15
+
+        # simple samples of different shapes to assert ErrorRaising
+        self.simple_x = np.array([[0, -np.pi, np.pi], [0, -np.pi, np.pi]])
+        self.simple_y = np.array([0, -np.pi, np.pi])
+        self.simple_z = np.array([0, np.pi, np.pi / 2, -np.pi])
+
+        # check for ground truth consistency with REAL/ARTIFICIAL LFP-dataset
+        # REAL LFP-DATASET
+        # Load first & second data file
+        filename1_r = os.path.sep.join(['cross_testing_scripts',
+                                        'lfp_dataset1.mat'])
+        dataset1_r = scipy.io.loadmat(filename1_r, squeeze_me=True)
+        filename2_r = os.path.sep.join(['cross_testing_scripts',
+                                        'lfp_dataset2.mat'])
+        dataset2_r = scipy.io.loadmat(filename2_r, squeeze_me=True)
+        # get the relevant values
+        self.lfps1_r = dataset1_r['lfp_matrix'] * pq.uV
+        self.sf1_r = dataset1_r['sf'] * pq.Hz
+        self.lfps2_r = dataset2_r['lfp_matrix'] * pq.uV
+        # ARRTIFICIAL LFP-DATASET
+        filename1_a = os.path.sep.join(['cross_testing_scripts',
+                                        'artificial_LFPs_1.mat'])
+        dataset1_a = scipy.io.loadmat(filename1_a, squeeze_me=True)
+        filename2_a = os.path.sep.join(['cross_testing_scripts',
+                                        'artificial_LFPs_2.mat'])
+        dataset2_a = scipy.io.loadmat(filename2_a, squeeze_me=True)
+        # get the relevant values
+        self.lfps1_a = dataset1_a['lfp_matrix'] * pq.uV
```

Comment:
The naming of the variables are using different conventions, i.e., in some `_a` or `_r` are used for naming the artificial or real datasets, while later we have `ARTIFICIAL` or `REAL`. The overall code would be more understandable if using the same naming and using full words.